playgrounds/firecamp-graphql/src/store/slices/change.slice.ts

```diff
@@ -29,7 +29,7 @@ interface IRequestChangeStateSlice {
    */
   disposeRCS: () => void;
 }
-//@note; always use _cloneDeep at its usage otherrwise it's value will be manipulate at global scope
+//@note; always use _cloneDeep at its usage otherwise its value will be manipulate at global scope
```

Comment:
this is a pair of unrelated corrections

`it's` = `it is`
`its` = possessive 
https://www.dictionary.com/e/its-vs-its/